### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Compwnter**
real

### CODE REVIEW

**Comment by dolfies** [comment]
Unreal.

File: README.md [hunk 1]
```diff
@@ -1,44 +1,37 @@
...
-took a shot at writing it again. It works.
-
-**This is "Litecord" / "litecord".** There are _no_ rewrites planned (for now :>).
+[litecord serviced]: https://gitlab.com/litecord
 
 ## Project Goals
 
 - Being able to unit test bots in an autonomous fashion.
 - Doing research and exploration on the Discord API.
-- Doing research on old clients and scrapped features.
```
Comment (lexisother) [inline README.md:29]: We do research on old clients.

File: README.md [hunk 2]
```diff
@@ -1,44 +1,37 @@
...
+
+- Being used as a "self-hostable Discord alternative".
 
 ## Caveats
 
 - Unit testing is incomplete.
 - Currently, there are no plans to support video in voice chats, or the
   Discord Store.
-- Many things are non-performant, deviate from the Discord API, are incomplete, etc.
-- Compatibility is preferred to accuracy. Because of this, many duplicate/deprecated fields exist in the API.
```
Comment (lexisother) [inline README.md:37]: This still holds true.

File: README.md [hunk 3]
```diff
@@ -51,12 +44,14 @@ Discord can implement new features at any time, for any reason. The following
 are not implemented, for example:
 
 - Threads
-- Auto moderation
-- Server boosts
```
Comment (lexisother) [inline README.md:55]: This also holds true.

File: README.md [hunk 4]
```diff
@@ -84,7 +79,7 @@ Optional requirement:
 ### Download the code
 
 ```sh
-$ git clone https://github.com/dolfies/patchcord.git && cd patchcord
+$ git clone https://gitlab.com/dolfies/patchcord.git && cd patchcord
```
Comment (lexisother) [inline README.md:82]: We don't have a repo on GitLab.

**Review by lexisother — CHANGES_REQUESTED** [review]

**Pull request closed without merging**